portfolio/src/main/webapp/index.html

```diff
@@ -8,41 +8,60 @@
+      </div>
+      <section class="left-content">
+        <h3>Fun Facts</h3>
+        <ul>
+            <li>I was born at home and my dad delivered me (a little different
+                to his day job as an accountant!)</li>
+            <li>I am the youngest of 4 children</li>
+            <li>My favourite food is black olives</li>
+            <li>My Myers-Briggs personality type is INFJ</li>
+            <li>I am a Rainbow GirlGuide volunteer</li>
+            <li>My favourite TV show is Gilmore Girls</li>
+            <li>I've been a member of Swing Dance society for 2 years</li>
+            <li>I have a 500+ streak on Duolingo</li>
+        </ul>
+      </section>
+      <section class="right-content">
+        <h3>Edinburgh</h3>
+        <p>Here is a sketch of mine, showing some of my favourite places in Edinburgh.</p>
+      </section>
+      <div class="img-nonprofile">
+        <img src= "/images/edinburgh-sketch.png" alt="Edinburgh Buildings Sketch" 
+        width=480px height=333px margin-right=15px>
+      </div>   
+      <section class="left-content">
+        <!-- Code from Original Template -->
+        <p>Click here to get a random message:</p>
+        <button onclick="addRandomGreeting()">Hello</button>
+        <div id="greeting-container"></div>
       </section>
-      <img src= "/images/profile.jpg" alt="Profile Picture" width=200px height=200px margin-right=15px>
-      <h3>FUN FACTS</h3>
-      <ul>
-          <li>I was born at home and my dad delivered me (a little different
-              to his day job as an accountant!)</li>
-          <li>I am the youngest of 4 children</li>
-          <li>My favourite food is black olives</li>
-          <li>My Myers-Briggs personality type is INFJ</li>
-          <li>I am a Rainbow GirlGuide volunteer</li>
-          <li>My favourite TV show is Gilmore Girls</li>
-          <li>I've been a member of Swing Dance society for 2 years</li>
-      </ul>   
-      <!-- Code from Original Template -->
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
       <!-- This is the links section -->
       <div class="buttonText">
-          <a href="https://www.linkedin.com/in/lornafarmstrong" target="_blank">
-          <button class="hyperlinked-button buttonBlue">LinkedIn</button></a>
-          <a href="https://www.https://github.com/lornaarmstrong" target="_blank">
-          <button class="hyperlinked-button buttonGray">GitHub</button></a>
+        <a href="https://www.linkedin.com/in/lornafarmstrong" target="_blank">
+        <button class="hyperlinked-button buttonBlue">LinkedIn</button></a>
+        <a href="https://www.https://github.com/lornaarmstrong" target="_blank">
+        <button class="hyperlinked-button buttonGray">GitHub</button></a>
       </div>
+      </section>
       <footer>lornaarmstrong</footer>
```

Comment:
Would be nice to leave a TODO comment on top of this footer to highlight that it's not currently displayed as expected and you will fix it later. [Example](https://www.jetbrains.com/help/idea/using-todo.html)